docs/manuals/source/Configuration/Director.rst

```diff
@@ -1151,21 +1151,42 @@ The directives within an Options resource may be one of the following:
 .. config:option:: dir/fileset/include/options/HardLinks
 
    :type: yes|no
-   :default: yes
+   :default: no 
+
+   New default since :sinceVersion:`23.0.0: Bareos >= 23.0.0`
```

Comment:
```suggestion
   Since :sinceVersion:`23.0.0: fileset keep hard links option switched from yes to no`
   the default is :strong:`no`, before it has been :strong:`yes`.
```
sinceVersion writes "Bareos >= xxx" automatically. The text after the column will be written to the index.

This information is now twice in this section. It might be enough, to combine this with the note at the end.

Is this information also provided as breaking (?) change in the changelog?